AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Request/FetchMFAPreferenceRequest.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+/// Request for fetching user MFA preferences
+public struct FetchMFAPreferenceRequest: AmplifyOperationRequest {
+
+    /// Extra request options defined in `FetchMFAPreferenceRequest.Options`
+    public var options: Options
+
+    internal init(options: Options) {
```

Comment:
The existing requests have a `public` init, but these new ones do not. 
Is this expected?

Reply:
I didn't see any value of making them public, because they were only used internally.. But changed it to match others.